lib/tasks/bundlers/generateComponentPreload.js

```diff
@@ -51,37 +60,63 @@ module.exports = function({workspace, dependencies, taskUtil, options: {projectN
 					`found for project: ${projectName}`);
 			}
 
-			return Promise.all(allNamespaces.map((namespace) => {
+			const allFilterExcludes = negateFilters(excludes);
+			const unusedFilterExcludes = new Set(allFilterExcludes);
+
+			const bundleDefinitions = allNamespaces.map((namespace) => {
 				const filters = [
 					`${namespace}/`,
 					`!${namespace}/test/`,
 					`!${namespace}/*.html`
 				];
 
-				// Exclude other namespaces
+				// Add configured excludes for namespace
+				allFilterExcludes.forEach((filterExclude) => {
+					// Allow all excludes and limit re-includes to the component namespace
+					if (filterExclude.startsWith("!") || filterExclude.startsWith(`${namespace}/`)) {
```

Comment:
Note: The second term only works because negateFilters(...) also normalizes filters, They no longer can start with a '+'.

Reply:
"Includes" coming out of `negateFilters` now always fancy a leading "+"